### DISCUSSION TITLE
Language example OC 4.0

### DISCUSSION THREAD

**Discussion by condor2** [opencart/opencart#11601] [General]
Based on German Language example from here https://github.com/opencart/examples/tree/main/oc_language_example I don't see nothing related with language files from 

https://github.com/opencart/opencart/tree/master/upload/extension/opencart/admin/language/en-gb

Or you need to copy all files **extension/opencart/admin/language/en-gb** to **oc_language_example/admin/language/de-de** ?

**Comment by mycroftsk** [comment]
Basically yes, your file structure should be like

`\extension\sample_language\admin\language\sl-sl\catalog`
`\extension\sample_language\admin\language\sl-sl\common`
`\extension\sample_language\admin\language\sl-sl\cron`

etc...

**Reply by condor2** [reply]
Yes...basically should be ....

\extension\sample_language\admin\language\sl-sl\
\extension\sample_language\catalog\language\sl-sl\

\extension\sample_language\extension\sample_language\admin\language\sl-sl\
\extension\sample_language\extension\sample_language\catalog\language\sl-sl\

**Reply by mycroftsk** [reply]
Seems like I misread you question 😅

I have same problem, translations for opencart buildin extensions is not loaded and I see only untranslated strings like `basic_heading_title`.

**Reply by condor2** [reply]
Yes. Maybe @danielkerr add built in extensions in oc_example_language

**Comment by mhcwebdesign** [comment]
Maybe it's an issue with a fallback to the default language (usually en-gb) when a text string isn't translated?
Have to test it.

**Reply by mycroftsk** [reply]
Any progress to this? Is this bug or how to properly translate extensions?
